docs/tutorials/integrations/kserve-mm/abn.md

```diff
@@ -28,69 +28,46 @@ This tutorial describes how to do A/B testing of a backend ML model hosted on [K
 
 ## Deploy the sample application
 
-A sample application using the Iter8 SDK is provided. Deploy both the frontend and backend components of this application as described in each tab:
+A simple sample two-tier application using the Iter8 SDK is provided. Deploy both the frontend and backend components:
 
-=== "frontend"
-    ```shell
-    kubectl create deployment frontend --image=iter8/abn-sample-mm-frontend-go:0.17.3
-    kubectl expose deployment frontend --name=frontend --port=8090
-    ```
-
-    The frontend component is implemented to call the Iter8 SDK method `Lookup()` before each call to the backend ML model. The frontend component uses the returned version number to route the request to the recommended version of the model.
-
-=== "backend"
-    The backend application component is an ML model. Deploy the primary version of the model using an `InferenceService`:
-
-    ```shell
-    cat <<EOF | kubectl apply -f -
-    apiVersion: "serving.kserve.io/v1beta1"
-    kind: "InferenceService"
-    metadata:
-      name: backend-0
-      labels:
-        app.kubernetes.io/name: backend
-        app.kubernetes.io/version: v0
-        iter8.tools/watch: "true"
-      annotations:
-        serving.kserve.io/deploymentMode: ModelMesh
-        serving.kserve.io/secretKey: localMinIO
-    spec:
-      predictor:
-        model:
-          modelFormat:
-              name: sklearn
-          storageUri: s3://modelmesh-example-models/sklearn/mnist-svm.joblib
-    EOF
-    ```
+### frontend
 
-    ??? note "About the primary `InferenceService`"
-        The base name (`backend`) and version (`v0`) are identified using the labels `app.kubernetes.io/name` and `app.kubernetes.io/version`, respectively. These labels are not required.
+The frontend component uses the Iter8 SDK method `Lookup()` before each call the backend (ML model). The frontend uses the returned version number to route the request to the recommended version of backend.
 
-        Naming the instance with the suffix `-0` (and the candidate with the suffix `-1`) simplifies describing the application (see below). However, any name can be specified.
-        
-        The label `iter8.tools/watch: "true"` is required. It lets Iter8 know that it should pay attention to changes to this application resource.
+```shell
+kubectl create deployment frontend --image=iter8/abn-sample-mm-frontend-go:0.17.3
+kubectl expose deployment frontend --name=frontend --port=8090
+```
 
-## Describe the application
+### backend
 
-In order to support `Lookup()`, Iter8 needs to know what the application component versions look like. A _routemap_ is created to do this. A routemap contains a description of each version of an application and may contain [routing templates](../../../user-guide/topics/routemap.md). To create the routemap:
+The backend application component is an ML model. Release it using the Iter8 `release` chart:
```

Comment:
To clarify, there isn't anything special about the backend component right? Only the frontend is using the Iter8 SDK? If that's the case, it might be nice to mention that here. We want to show that we designed the SDK to be as unobtrusive as possible.

Reply:
did so in the introduction to the simple sample application a few lines above